BookingQueueSubscriber/BookingQueueSubscriber.UnitTests/MessageHandlers/ParticipantsAddedHandlerTests.cs

```diff
@@ -19,7 +19,7 @@ public class ParticipantsAddedHandlerTests : MessageHandlerTestBase
         [Test]
         public async Task should_call_video_api_when_request_is_valid()
         {
-            var messageHandler = new ParticipantsAddedHandler(VideoApiServiceMock.Object);
+            var messageHandler = new ParticipantsAddedHandler(VideoApiServiceMock.Object, VideoWebServiceMock.Object);
```

Comment:
Should these tests be updated to verify that video web has been called as well?

Reply:
Yes. I've added it